tools/docker-compose-dev-datasets-server.yml

```diff
@@ -43,20 +45,29 @@ services:
     depends_on:
       - mongodb
     restart: always
-    network_mode: host
+    ports:
+      - ${ADMIN_UVICORN_PORT-8081}:${ADMIN_UVICORN_PORT-8081}
```

Comment:
are you reverting these changes?

Reply:
Yes, `network_mode: host` doesn't work on mac :/
But keeping the ports is fine